src/components/Map.js

```diff
@@ -13,8 +15,11 @@ import './Map.scss';
 export default function Map() {
   const newQuery = decodeURI(window.location.search).split('=')[1];
```

Comment:
```suggestion
  const location = useLocation();
  const newQueries = new URLSearchParams(location.search);
  const type = newQueries.get('type')
```

useLocation() 안에는 search라고 해서 query 값만 따로 얻을 수 있는 프로퍼티가 있습니다.
이 프로퍼티를 new URLSearchParams(location.search) 이 형태로 넣으면 get 을 써서 얻을 수 있습니다. 아까 전달드린다는 거를 깜빡하고 전달하지 않았군요..

![image](https://user-images.githubusercontent.com/92532339/173418715-73833438-d1c0-4690-90f5-e616a1a1b57a.png)

![image](https://user-images.githubusercontent.com/92532339/173419292-fb98cbdf-7b5d-445f-9d2f-d3be259f5625.png)

![image](https://user-images.githubusercontent.com/92532339/173419371-cd1349dc-0bd5-44f5-ac46-e7dd3de5b531.png)